Kernel/Arch/x86/ProcessorInfo.h

```diff
@@ -20,7 +21,7 @@ class ProcessorInfo {
     Processor& m_processor;
     String m_cpuid;
     String m_brandstr;
```

Comment:
Can we make these `KString` as well?

Reply:
Yes, bit I chose to carve off this piece for now. There are tons of things to fix, and not so much time. 😄